### PR TITLE
Removes Space Hotel

### PR DESCRIPTION
**Pull request by dinnerwoerror**
## About The Pull Request

removes the space hotel

## Why It's Good For The Game

bro who actually uses the space hotel

## Changelog
:cl:
removes the space hotel
/:cl:

### CODE REVIEW

**Comment by triplezeta** [comment]
good PR

**Comment by dinnerwoerror** [comment]
also zeta should i add ball gags

**Comment by triplezeta** [comment]
do it

**Comment by SynnGraffkin** [comment]
Add collars too, you kinky nerd

**Comment by dinnerwoerror** [comment]
alright bitch boy you told me to

**Comment by TheNeoGamer42** [comment]
the damn hotel SHOULD be removed but uh.... why the fuck are ALL the checks pretty much failing, the important 3 atleast

**Comment by triplezeta** [comment]
the answer is shitcode

**Comment by TheNeoGamer42** [comment]
yote

**Comment by dinnerwoerror** [comment]
the only people downvoting this pull request are power gaymur spes esplourers who burgle the place for le ebin taser shotgun

**Comment by MarkSuckerberg** [comment]
Ironically we could also just put the space hotel into the ruin blacklist instead of just outright removing it

**Comment by dinnerwoerror** [comment]
IrOnIcAlLy We CoUlD aLsO jUsT pUt ThE sWaRmEr RuIn InTo ThE bLaCkLiSt InStEaD oF jUsT oUtRiGhT rEmOvInG iT

**Comment by MarkSuckerberg** [comment]
this is true

**Comment by MarkSuckerberg** [comment]
thank you for providing much needed insight to my internal decision-making process, ogan

**Comment by dinnerwoerror** [comment]
i prefer doing it in such a brutish and pasquinadish format that i don't even need to make a decision

**Comment by MarkSuckerberg** [comment]
truly we could all stand to learn from your methods

**Comment by dinnerwoerror** [comment]
there was a time when i was a facility guard on SCP secret laboratory and i literally thumped my chest while doing caveman ooking over the intercom and then i killed 8 chaos insurgency

File: code/game/area/areas/ruins/space.dm [hunk 1]
```diff
@@ -63,71 +63,7 @@
...
-	name = "Hotel Shuttle Dock"
-	icon_state = "start"
-
-/area/ruin/space/has_grav/hotel/workroom
-	name = "Hotel Staff Room"
-	icon_state = "crew_quarters"
-
-
-
+	ambientsounds = list('sound/ambience/ambivapor1.ogg'
```
Comment (MarkSuckerberg) [inline code/game/area/areas/ruins/space.dm:66]:
```suggestion
	ambientsounds = list('sound/ambience/ambivapor1.ogg')
```

**Comment by MarkSuckerberg** [comment]
your methods are brash but effective

**Comment by dinnerwoerror** [comment]
now this is podracing

**Comment by SynnGraffkin** [comment]
@MarkSuckerberg and he'll just do the same mocking tone on your next PR.

**Comment by MarkSuckerberg** [comment]
unironically I'm not particular opposed to him talking like this, he actually made me think about if I should actually close this PR or not

**Comment by MarkSuckerberg** [comment]
ogan you do realize you need to remove the actual map right

**Comment by dinnerwoerror** [comment]
yea totally

**Comment by SynnGraffkin** [comment]
Much like the swarmer debaucle, it's much easier to just put it on a blacklist so if situations ever change, it can be easily reenabled.

**Comment by dinnerwoerror** [comment]
yep mhmm i knew about the map removing all along

**Comment by MarkSuckerberg** [comment]
yolo

**Pull request merged**